protocol-units/bridge/move-modules/sources/atomic_bridge_counterparty.move

```diff
@@ -15,6 +15,8 @@ module atomic_bridge::atomic_bridge_counterparty {
     const COMPLETED: u8 = 2;
     const CANCELLED: u8 = 3;
 
+    const EINCORRECT_SIGNER: u64 = 7;
```

Comment:
feat branch has several error codes, errors are also not named, eincorrect_signer is now error 1. Might be good to get the changes from feat branch.

Reply:
Thanks; I've now adjusted the codes to be consistent with the ones in feature branch and use named errors throughout.